holoviews/plotting/util.py

```diff
@@ -451,6 +453,31 @@ def get_min_distance(element):
     return 0
 
 
+class univariate_composite(Operation):
+
+    output_type = Area
+
+    def _process(self, element, key=None):
+        plot_opts = Store.lookup_options(Store.current_backend, element, 'plot').kwargs
+        bw = plot_opts.pop('bw', univariate_kde.bandwidth)
+        transformed = univariate_kde(element, bandwidth=bw)
+        Store.transfer_options(element, transformed, ['bw'])
+        return transformed
```

Comment:
Transferring styles from input to output seems like it should be an option for the ``Compositor`` when you register the operation.

Reply:
Do you want that to hold up merging this PR?